### DISCUSSION TITLE
[BUG]Connect-PnPOnline with clientId and Secret value

### DISCUSSION THREAD

**Discussion by Mohamedbg23** [pnp/powershell#3067] [Q&A]
i've registered my app and created the certifications and generated the secret value and even i give the api permission to my app i can't access to anything
![Capture](https://user-images.githubusercontent.com/123561528/234919126-2c329900-0ae4-45ba-bd10-128a1a3a6e33.PNG)
![Capture1](https://user-images.githubusercontent.com/123561528/234919330-4620b944-fd70-470a-985f-7c26fcdd292f.PNG)
![Capture2](https://user-images.githubusercontent.com/123561528/234919919-3c6e746f-88bd-49a1-bca1-87096c4b1f7f.PNG)

**Comment by yesh-rajawat-robomq** [comment]
Any update on this issue?

**Comment by jefmeyer** [comment]
Facing the same issue.  Any updates?
